docs/pages/access-controls/access-request-plugins/ssh-approval-slack.mdx

```diff
@@ -193,6 +202,23 @@ This should result in a config file like the one below:
 ```toml
 (!examples/resources/plugins/teleport-slack.toml!)
 ```
+</TabItem>
+<TabItem label="Helm Chart">
+The Slack Helm Chart uses a YAML values file to configure the plugin.
+On the host where you've helm installed, create a file called
+`teleport-slack-helm.yaml` based on the following example:
```

Comment:
I'm not sure what we're asking the reader to do here. Is this instructing the reader run the `teleport-slack configure` command, then manually edit it to be a Helm values file?

Also, the text says "This should result in a config file like the one below", but the one below looks very different from the result of the `teleport-slack configure` command.

Reply:
Ooops, I failed to delete the old lines.